.github/ci-scripts/bash/check_approval.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+check_commits() {
```

Comment:
In some files we are following `camelCase` for function name while `underscore_separated` in some places. Let us stick to `camelCase` everywhere